ynr/apps/elections/templates/elections/sopn_for_ballot.html

```diff
@@ -0,0 +1,83 @@
+{% extends "base.html" %}
+{% load pipeline %}
+{% block content %}
+
+  <style>
+    .pdf_container:not(:empty) {
+      border:1px solid #EEE;
+      max-width:100% !important;
+      width:90em !important;
+    }
+    .page_container .text_layer {
+      position: absolute;
+      left: 0;
+      top: 0;
+      right: 0;
+      bottom: 0;
+      overflow: hidden;
+      opacity: 0.2;
+      line-height: 1.0;
+    }
+
+    .page_container .text_layer > div {
+      color: transparent;
+      position: absolute;
+      white-space: pre;
+      cursor: text;
+      transform-origin: 0% 0%;
+    }
+    .page_container .text_layer > div.highlight{
+      color: black;
+      background-color: yellow;
+      border:1em solid yellow;
+      margin:-1em;
+      position: absolute;
+      white-space: pre;
+      cursor: text;
+      transform-origin: 0% 0%;
+    }
+  </style>
+
+  <h1>Statement of Persons Nominated for <a href="{{ object.get_absolute_url }}">{{ object.post.label }} {{ object.election.election_date }}</a></h1>
```

Comment:
Can we include the election name here as well as the post name. Also this section feels like the layout is a bit odd. I think we've also got a bit too much text in a `<h1>` (and that will be worse once we clarify with the election title). Putting the button on the next line down/in a new block element will also help

![screenshot at 2019-02-08 16-29-37](https://user-images.githubusercontent.com/6025893/52494630-65073180-2bc6-11e9-9cca-8a2d7745f994.png)